test/integration/scheduler/queue_test.go

```diff
@@ -105,10 +105,6 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			t.Fatalf("Expect Pod %v to fail at scheduling.", podInfo.Pod.Name)
 		}
 		testCtx.Scheduler.Error(podInfo, fitError)
-
```

Comment:
well explained and looks good for me.

BTW the changed file is an integration test file, not e2e test file.
It is necessary to update the title and description of this pull request.